demo/styles.css

```diff
@@ -1,9 +1,18 @@
 html {
 	font-size: 20px;
 }
```

Comment:
Could these styles go into another demo component that people could use to wrap their demos? i.e. `<d2l-demo-page>` or something.

Reply:
Oh interesting.  Justin just created this [html-include-element](https://github.com/justinfagnani/html-include-element) that could potentially be used to reduce some of the repeated page HTML for our component demo pages and visual-diff pages.  I wouldn't mind chatting/thinking more on this.